test/resources/malformed.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0"?>
 <root>
-  <note><![CDATA[
-    This is  a note!
-  ]]></note>
+  <note>complete note</note>
+  <note>incomplete note
```

Comment:
This change was made to ensure that tests relying on malformed XML did not start failing as soon as control chars did not constitute a malformation.